<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmDevicePanel</class>
 <widget class="QWidget" name="frmDevicePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>168</width>
    <height>96</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="gboxDeviceTitle">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="ButtonDevice" name="btnDevice" native="true">
           <property name="minimumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labDeviceName">
           <property name="text">
            <string>温湿度设备1</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ButtonPower" name="btnPower" native="true">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="gboxDevicePanel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>3</number>
         </property>
         <item row="0" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout">
           <property name="horizontalSpacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="ButtonPilot" name="btnTemp" native="true">
             <property name="minimumSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="labCurrentTemp">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>温度 :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="ButtonPilot" name="btnDamp" native="true">
             <property name="minimumSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="labCurrentDamp">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>湿度 :</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="labCountNoRead">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>未读 : 0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labCountAll">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>总条数 : 0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonPilot</class>
   <extends>QWidget</extends>
   <header location="global">usercontrol/buttonpilot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonDevice</class>
   <extends>QWidget</extends>
   <header location="global">usercontrol/buttondevice.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonPower</class>
   <extends>QWidget</extends>
   <header location="global">usercontrol/buttonpower.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
